model/common/src/icon4py/model/common/interpolation/rbf_interpolation.py

```diff
@@ -346,7 +346,7 @@ def index_offset(f):
             rbf_vec_coeff_np[j][i + horizontal_start, valid_neighbors] = sla.cho_solve(
                 z_diag_np, rhs_np[j][i, valid_neighbors]
             )
-    rbf_vec_coeff = tuple([array_ns.asarray(x) for x in rbf_vec_coeff_np])
+    rbf_vec_coeff = [array_ns.asarray(x) for x in rbf_vec_coeff_np]
```

Comment:
This other way would have kept the original `tuple` type annotation, and since it seems the list is going to be converted to a tuple anyway, probably makes more sense.
```suggestion
    rbf_vec_coeff = tuple(array_ns.asarray(x) for x in rbf_vec_coeff_np)
```

Reply:
> are you aware that tuple[NDArray, ...] also express arbitrary-sized tuples?

I was not! Thanks. In that case I like the option here (without `@overload`, with tuples). Let me try that as well.